scripts/globals/weaponskills.lua

```diff
@@ -959,3 +959,61 @@ function takeWeaponskillDamage(defender, attacker, params, finaldmg, slot, tpHit
 
     return finaldmg;
 end
+
+function applyAftermathEffect(player, tp, spower_1, spower_2, spower_3)
+    spower_1 = spower_1 or 1
+    spower_2 = spower_2 or 1
+    spower_3 = spower_3 or 1
+
+    -- AFTERMATH LEVEL 1
+
+    if ((tp >= 1000) and (tp <= 1100)) then
```

Comment:
they are linear in their 1000 ranges (1000-2000, 2000-3000).  They jump between ranges because the different tiers of aftermaths have different effects.  However, I would like to point out that they are not always the same values (melee weapons AM2 is 20-60 attack, but tupsimati is 20-30 MAB)